src/main/java/Main.java

```diff
@@ -13,22 +13,30 @@
     private static CookingOven cookingOven;
     private static WaterHeater waterHeater;
     private static BufferedReader br;
+    private static String filePath = "/projects/assignment02/src/main/input.txt";
```

Comment:
Never have any hardcoded strings in code. Try to use properties file or get the values from commandline arguments.